internal/testutil/pkgdata.go

```diff
@@ -13,150 +12,79 @@ import (
-x84GDHOGDCBbswHli/o+dT5Bf4BAZgOkC2SA5/3cMgO0x4D7XZbZEI4xG+Ao+oLzGHDH5g+wGeEz
-/CGIum9bDCLsDkpC2HBHSfj5isB5xf3jwxpwCShbRR0ZJMZRmtwm4y6oBnpA13Te7gZ4GbUi1hGT
-sP0QiLEQl5J4YWCZ0FGACiYTUnrq32kVDXuAtCJwiJ4QVOtxVLIBV9EZgQDIWQ7KQFSrnM0GqFPq
-Obmty/XDYcBS/7+RAUWz+R5552wA5FOOAW59wABF00oiSkxbtQvmbQgX15D6sUkQ3VoOj1zfuYaw
-0QqT0Eb5KFqueIDH5lQevf8aH0fMHCQZHyD2sVG/SEcx+3NRIdWfkn04FQp2uSdEwxmRk6CAIb1G
-6xcAGrZgTsEVa5zzc2vZeg5pXGe0zwCQIAvFdj+jHtT+83DoQO6jGVqfd/Yrf5rC3F4Hkr+8qYQJ
-YXSf+d33nIZqQPYBfLo162MU/ANZFb1vwYGvtpkimMkyffzsgKbk41GBIa6IpPmXAHJ1puwTI/Bj
-B7bIrqiAP/eNp+SJU+8FjZ6MZ7ngKtifGp/0RiG3MT7qYB+S8DvHnK+clckBc44JUcjwGqJS6nhK
-2LlEixEegC/zCWf2Aig2FrKJrAWFR4d9PLNWoB1NTS38TxAAqOokA8+ZCTJVC0Oqvwd5LXQva7qQ
-FS1En6Avjs9lQRZz1abZmIglx3WtBMfaNC1IOdqLEE63B91jCuGqQUKFXJcpDtc+Mrg3uk/ZXPWi
-GnQOgMyNeUoZ1OOxf50FEXIL5c2CYv9VAo+KDPEzVSvTivS+EuQyd9d/9/0DUYGfjKQUqoobEC9b
-UH4v2GRBPBjLxUGlL8J6KwxXhWgjpdX/O48zPE1Bb/3cxIhQaEWRlpSrwhI7fPkLWhOBvijulQmD
-2NMNiuWX18EqSCmKsnqM2+EgEFu1DFaiS/WTdSb0vzzVtxq7AjwiiSIF/nktYwdcPN2DYXHtKZ2N
-/31ZMhMWcjf6ayFQDizjnYPAamYx4Jb8i84oJT6waB/nyHUkhMBMieJs+9J3cO9xI0YOnIAC7JAS
-qUjG6wd4Cf3v1fOy3DbZgmviroh/YMpWxWptgWzrdmW//eHdHSzZrEmGZpsQmJvHIgBs13/+j+4u
-0wV3OphfQnPCZJf8Rw0x1AM/AmX37GEno2HCfvI26MwTl1QrxHalg1ckqoWnLrLj6DIhlQMUYHNf
-ps57u05gdJj2o1i42SJm13/5pxefld6J/VippHJT+YrIdtj/tZXUFUixI6jOliJAjGH6qG1IDp6w
-6rSH2wIltPWe4j/ztdYy3Wcc3zyNYZMMT6oL8wMFXOuLVttkDCVC9GrOdSkyirbgY8yEfkBmUwoj
-UURaO/DxEZoVoKzhc9vALyBY/QjHlRzPSSsN5TbAdLETZf6rgM5a3a4Qkq0LlP3fbZOewo05YGPi
-2oUlmp6cIPGXqdrL8QR/e7g0qREMJQseUgp+HUpzxvt7Lfosjtwe9u2IAmMowXN5ekceb89ram9+
-mxQ2IR5qJfT2U4DnOjFA1PniK/R7WXPCh84g2E2go95x4AdGaKJ/HXxM1z2A3IrQuUQCoAHpLkxh
-SrCzPMou5IPI8/Oyq+UvUszGq14t7WoCQatGdDGOaY/HW+iMDTFMPRNrW1V4Y8y9FYcIYbrYWsfS
-nFi9lG8FvSDYhsUQfqZ/K/AUn9GK3YpHA+iTwTOZoRLesFWgsRVwzL1mFSsC1v8qBRFvTdmTcuc6
-AOarS446c71VWYUOwOEkxDvEj87+D/qR9f8W7yNBA1GWkyyUrqwS7yZyGm43iPRrw9vup7hs51lN
-tNs2GS8JVI4x12TsyvviAkeOnkOtaPlc2dAlnYgHCiAieJc11w6A6kF7nTOLmgbgpJ0vgWhDX6R5
-GxTSuTS9IJz7KoeRPugOUVb7APLc83k6cNfdjSFiRxbwmqrAs+fmRFm7jQuMTt0m9abmWq3C4oAB
-Tki+4JSWJEpdUxHtZWOwZkrmPakPvSqfmlMsIH+VeYAIVQQSg0z370BcC8nONv/GLY3Abp/E3dnh
-ceWRpGMVbKcPdhAxVEPnKicIM1FZUkeWsW9IG8fTCBUOP0vQXCA2zpne+hp+SE/FwYqYlCc8n+ko
-PzBuPEfYcXyBfR+bfv/Its4vgJOLw4EVpUr9JSPnQfL4ZtooI+oRAztEoQEk4T2QOKEknA6f+wz8
-614xl2bUX6oQ8IzX7W0gMgD+naSATynZ/zGATrFh1gMYNRuwK8WALqZ56cWAye0BDGHL/eR1+D+S
-fmbN3bI6I1l0TW3S6JE2AMxkSyQ2xtcGEBqrGNDx7koM0CQtiSEMiGJfEtEMD3DD8gEqZ7xCbK5a
-NbWjB9ZbqjMGNDbZADvj/pnDAZXFhjoHIK1kTo+EtfnoGAzLJLm6CJLw6SXGjKb1TQXeQJLnl45+
-dYiMSUn6m9CRdG1QGWGkEvvSAmzgsWQDboABkB3AD3HJ+dsigMkGdILs/WiKAwc9AHZFcI5Fg5O0
-/xmyAfifAzCJjEd5BKZ1gLK4sb3eItKNr7mAaa3dCb+DkHsMbZphWlmBlMNTA0hlHIHOa4Z7BAQ7
-BIsAGAJD+LGgIzDiD2ntudH6JU80wD0rklyRGBV2dEx8VJG5LVwpcXb8PI0lMdi+wwUPoBkQGEIN
-7UwLQKMDjg8YkdmAmRrEgCSRRhMWBgBBSuI7S4BVs7zODfj1ja0TF+KbjdQDGANIcRlZhwzQSlla
-abj2Z7KoYMBXz9dwNNP2Aw13FguKkogezW5cqy4lCg==
-`
+func init() {
+	PackageData["base-files"] = MustMakeDeb(baseFilesPackageEntries)
```

Comment:
I wonder if we should keep calling it `base-files` since it contains file not found in traditional `base-files` packages. For example, the `/usr/bin/hello` script. And also lacks a few files (and dirs) from the usual base-files packages.

Since we are refactoring, how about renaming the entry to something like `test-package`?

Reply:
There has been a change of plan, we are no longer going to change base-files but the tests instead. Gustavo had a good point about the previous version of the PR not being the best solution, either we use a real package or we create tests data specifically.